app/controllers/admin/petitions_controller.rb

```diff
@@ -7,6 +7,13 @@ def index
     else
       petitions_by_filter_or_keyword_search
     end
+
+    respond_to do |format|
+      format.html
+      format.csv {
+        send_data PetitionsCSVPresenter.new(petitions), disposition: "attachment; filename=petitions.csv"
```

Comment:
I think we can just use the instance variable here and get rid of the `attr_reader`